.github/workflows/test.yml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
       - master
+      - github_actions
```

Comment:
Is there any reason to limit branches that run tests at all? It results in ugly hacks like what you had to do in this commit.

Reply:
This is just because github actions is not yet enabled in this repository, otherwise CI would've run because of the pull_request event. The reason to limit branches is to avoid duplicate CI runs on pushes to pull requests.

Reply:
Even after GitHub Actions is enabled, user might want to run tests *before creating a PR*.

Reply:
Yes. This is maintainers preference. My preference is that users either open a WIP PR or modifiy the workflow temporarily, and avoid all the duplicate builds by default. Others may have other preference. It's no big deal, I can change this to whatever @drbrain prefers.